htdocs/accountancy/admin/subaccount.php

```diff
@@ -393,7 +393,7 @@
 		// Subaccount label
 		if (!empty($arrayfields['label']['checked'])) {
 			print "<td>";
-			print $obj->label;
+			print $obj->nature == 2 ? '<span style = "color: red;"><b>'.$obj->label.' ('.$langs->trans("	Prospect").')</b></span>' : $obj->label;
```

Comment:
Parameter into langs->trans contains a non expected space.

Reply:
Thanks, I fixed that